tests/integration_tests/kafka_simple_include_start_ts/run.sh

```
@@ -0,0 +1,128 @@
+	kafka_dump --topic "$TOPIC_NAME" --timeout 90s --until-table cdc_start_ts_probe --until-count 2 >"$dump_file"
+	assert_probe_start_ts "$dump_file" "$ts" "$expect_field"
+
+	cdc_cli_changefeed remove -c "$changefeed_id"
+}
+
+function run() {
+	if [ "$SINK_TYPE" != "kafka" ]; then
+		return
+	fi
+
+	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)